CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## 1.8.6 (Upcoming)
 
+### NOTES:
+* hcp: users of HCP Packer will see some changes in how names are displayed during a
+       packer build for JSON templates. Previously only the name or the type were
+       displayed if both were available, and now both will be displayed. This does
+       not impact the behaviour of options such as only or except, they will continue
+       to work as they did before.
```

Comment:
I was under the impression that all users, regardless of HCP settings, will see a change in the output of the name. Is that not the case? 

If it is then I suggest a slight change to the note.

```suggestion
* core: Users will see some changes in how names are displayed during a Packer
     build for JSON templates. Previously only the builder type or the builder
     name, if it was set, would be displayed. Now for named builders
     (`"name":"mybuilder"`) the builder'ss type and name will be displayed (i.e
     "<type>.mybuilder". This does not impact the behavior of options such as
     only or except, they will continue to work as they did before.)
```

Reply:
Looks good to me, I'm integrating the change